### PR TITLE
Fixed formatting for embedded Terraform code within rst

### PR DESCRIPTION
**Pull request by dewan-ahmed**
# What changed, and why it matters

Changes made thanks to @mhoffm-aiven and [his work](https://github.com/MichaHoffmann/terrafmt/tree/mhoffm-add-restructured-text-support).

### CODE REVIEW

**Comment by dewan-ahmed** [comment]
I see that the formatting changes have caused the Terraform syntax highlighting to disappear 🤔 

I'll dig into this tomorrow.

File: docs/tools/terraform/reference/cookbook/kafka-flink-integration-recipe.rst [hunk 1]
```diff
@@ -30,75 +30,75 @@ For this, you'd like to run an Apache Flink job and write the filtered messages
...
     kafka_topic    = aiven_kafka_topic.source.topic_name
-    schema_sql = <<EOF
-      hostname STRING,
-      cpu STRING,
-      usage DOUBLE,
-      occurred_at BIGINT,
-      time_ltz AS TO_TIMESTAMP_LTZ(occurred_at, 3),
-      WATERMARK FOR time_ltz AS time_ltz - INTERVAL '10' SECOND
-    EOF
+    schema_sql     = <<EOF
```
Comment (mhoffm-aiven) [inline docs/tools/terraform/reference/cookbook/kafka-flink-integration-recipe.rst:78]: please try "<<-EOF"; pygments has a bug parsing heredoc comments: https://github.com/pygments/pygments/issues/2162

File: docs/tools/terraform/reference/cookbook/kafka-flink-integration-recipe.rst [hunk 2]
```diff
@@ -30,75 +30,75 @@ For this, you'd like to run an Apache Flink job and write the filtered messages
...
   }
-
+  
   resource "aiven_flink_table" "sink" {
     project        = var.project_name
     service_name   = aiven_flink.flink.service_name
     integration_id = aiven_service_integration.flink_to_kafka.integration_id
     table_name     = "cpu_high_usage_table"
     kafka_topic    = aiven_kafka_topic.sink.topic_name
     schema_sql     = <<EOF
```
Comment (mhoffm-aiven) [inline docs/tools/terraform/reference/cookbook/kafka-flink-integration-recipe.rst:94]: same here

File: docs/tools/terraform/reference/cookbook/kafka-flink-integration-recipe.rst [hunk 3]
```diff
@@ -108,17 +108,17 @@ For this, you'd like to run an Apache Flink job and write the filtered messages
       aiven_flink_table.sink.table_id
     ]
     statement = <<EOF
```
Comment (mhoffm-aiven) [inline docs/tools/terraform/reference/cookbook/kafka-flink-integration-recipe.rst:110]: same here

File: docs/tools/terraform/reference/cookbook/kafka-flink-integration-recipe.rst [hunk 4]
```diff
@@ -30,75 +30,75 @@ For this, you'd like to run an Apache Flink job and write the filtered messages
 
 ``services.tf`` file:
 
-.. code:: terraform
+.. code:: bash
```
Comment (mhoffm-aiven) [inline docs/tools/terraform/reference/cookbook/kafka-flink-integration-recipe.rst:33]: lets try to make terraform work by working around pygments

**Review by mhoffm-aiven — CHANGES_REQUESTED** [review]
see comments

File: docs/tools/terraform/reference/cookbook/kafka-flink-integration-recipe.rst (hunk 1 above)
Comment (mhoffm-aiven) [reply docs/tools/terraform/reference/cookbook/kafka-flink-integration-recipe.rst:78]: stripping the leading whitespace from the closing EOF is the proper way towrite it, technically pygments is spec conform there!

**Comment by dewan-ahmed** [comment]
@mhoffm-aiven I tested producing messages to the source Kafka topic and the expected filtered messages show up in the sink topic. This validates that the formatted terraform code from the doc still works.

**Comment by dewan-ahmed** [comment]
I see that the _toc.yml is missing two recipes that have been merged since. I'll wait before merging as I add one more commit to fix the _toc.yml

**Pull request merged**